models/repo_list_test.go

```diff
@@ -60,10 +56,152 @@ func TestSearchRepositoryByName(t *testing.T) {
 		Page:     1,
 		PageSize: 10,
 		Private:  true,
-		Searcher: &User{ID: 14},
 	})
 
-	assert.NotNil(t, repos)
 	assert.NoError(t, err)
 	assert.Equal(t, int64(3), count)
+	assert.Len(t, repos, 3)
```

Comment:
Add existing tests to testCases.

Reply:
I don't want to do it in this (#2371) PR because:
1. I want to show my PR will pass existing tests without changes
2. For moving existing tests to my added test cases it would be needed fixes at least in "repository.yml" and "access.yml".